index.html

```diff
@@ -286,6 +286,13 @@ <h2 class="section-title">SCHEDULE</h2>
                               </div>
                               <div class="col-2 col">
                                 <p class="speakers">Jiaqi Liu</p>
+                                <ul class="qcSchSpeakers clearfix">
+                                  <li>
+                                    <a href="#" class="tips" title="JIAQI LIU">
+                                      <img src="./assets/img/speakers/jiaqi-liu.jpg" alt="Jiqia Liu headshot" />
```

Comment:
clicking on the little photos in the schedule takes you to the top of the page, which is really weird